src/ai/backend/manager/api/vfolder.py

```diff
@@ -82,6 +84,60 @@
 VFolderRow = Mapping[str, Any]
 
 
+def vfolder_status_check(allowed_status: VFolderOperationStatus):
```

Comment:
It is good to implement wrapper method.
How about rename the method and the parameter?
Because I could not understand what this method actually does at the first glance.
Such as `vfolder_filter_by_status()`? This is just an example, so please do not use it if you do not think the name represents what this method does.